.github/CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# Copied files:
+
+.github/dco.yml @IBM/cloudant-sdks
+scripts/setup_couch.sh @IBM/cloudant-sdks
+scripts/setup_wiremock.sh @IBM/cloudant-sdks
+stubs/mappings.json @IBM/cloudant-sdks
+KNOWN_ISSUES.md @IBM/cloudant-sdks
+Jenkinsfile @IBM/cloudant-sdks
+
+# Generated code files:
+
+cloudant/v1.ts @IBM/cloudant-sdks
+test/unit/cloudant.v1.test.js @IBM/cloudant-sdks
+
+# Bump version files except the README:
+
+.bumpversion.cfg @IBM/cloudant-sdks
+package-lock.json
+package.json
```

Comment:
```suggestion
package-lock.json @IBM/cloudant-sdks
package.json @IBM/cloudant-sdks
```

Reply:
Thanks, rebased the file with the suggestion in c1e6c2e.